### ISSUE TITLE
export to pdf

### ISSUE THREAD

**Issue by tarunsingh5** [KhalisFoundation/sttm-desktop#63]
STTM2 provided ability to open the entire shabad in a window or PDF. Provide ability to split the display similar to STTM2 website.

**Comment by ManjotS** [comment]
- [ ] Provide Full Shabad View (like website), with toggle in display settings
- [ ] Add Split View (like website)
- [ ] Add export to PDF function

**Comment by navdeepsinghkhalsa** [comment]
Should full Shabad view be limited to exporting and not be an option in the app as a view option? I personally think so. Split view doesn't really make sense in this presentation app.

@inderpreetsingh We shouldn't build this into the webview/BrowserWindow display and then export, but rather at the app level. Please keep in mind that we have the power of node and shouldn't feel limited by what's possible in the browser.

**Comment by inderpreetsingh** [comment]
I think we still need to show some print preview, to give some feedback to user on what it's going to print and how it would look like. User might wanna change display settings before printing. It would be hard to fathom from slide view on what split view would look like in print.

What way do you suggest on app level? using `.printToPDF` seemed really simple and official way to print pdf of window content. I am not that well versed in best practices in electron. Let me know and I will add commits to this PR.

**Comment by navdeepsinghkhalsa** [comment]
I agree that we need to show a print preview, but I don't think the main or presenter windows are the places to do it. Let's open up a new window specifically for print preview, show the options on the left and the preview on the right. As the options are toggled on the left, the preview updates on the right. Here's an example and the PDF library that we can use to build the PDF.
http://pdfmake.org/playground.html
If you would like, I can set you up in your branch with a baseline of how to do it and you can continue it, or we can set up a chat or discuss here. Let me know!

**Comment by tarunsingh5** [comment]
is this in 4.0.2? @navdeepsinghkhalsa @inderpreetsingh

**Comment by navdeepsinghkhalsa** [comment]
@tarunsingh5 It is not

**Comment by saintsoldierx** [comment]
Is this in Dev or not @navdeepsinghkhalsa ? If not, can it be reassigned please

**Comment by tarunsingh5** [comment]
@maneetpaul @inderpreetsingh was this ever launched?

**Comment by maneetpaul** [comment]
Not to my knowledge @tarunsingh5 - will circle back to this

**Comment by prebhdevsingh** [comment]
@saintsoldierx can you provide a lofi design for this option?
